### PR TITLE
Release notes for week of 3 August 2015

### PR DESCRIPTION
**Pull request by srpearce**
@mhoeber Please review.

@explorerleslie Adding you to the review list because there are a lot of TNL bug fixes - let me know if you want to delete any or if you have any to add.

Thanks!

### CODE REVIEW

File: en_us/release_notes/source/2015/lms/lms_0804_2015.rst [hunk 1]
```diff
@@ -0,0 +1,29 @@
+
+====================================================
+Bug Fixes
+====================================================
+
+EdX has fixed the following bugs in the LMS.
+
+* Learners in two edX courses could not access their **Progress** page. This
+  problem has been resolved. (TNL-2879)
```
Comment (explorerleslie) [inline en_us/release_notes/source/2015/lms/lms_0804_2015.rst]: Remove this.

File: en_us/release_notes/source/2015/lms/lms_0804_2015.rst [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  from seeing courseware. (MA-1063)
+
+* When a learner earned partial credit on a problem, the score at the top of
+  the problem was not rounded to the nearest 10. For example, the score might be
+  ``0.666666666667/2 points``.
+
+* When a course team sent an email that contained a link for a course that was
+  not yet live, learners received a 404 "Not found" error message when they
+  selected the link in the email or followed another link to the course. Links
+  to non-live courses now redirect to the student dashboard. (TNL-2693)
```
Comment (explorerleslie) [inline en_us/release_notes/source/2015/lms/lms_0804_2015.rst]: Move this to top of list.

File: en_us/release_notes/source/2015/studio/studio_0804_2015.rst [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+====================
+Problem Improvements
+====================
+
+EdX has made the following improvements to problem components in Studio.
+
+* For drag and drop and Custom JS input problems, the **Show Answer** button
+  does not work correctly. EdX has changed the default **Show Answer** value
+  for these problems to **Never**.
```
Comment (explorerleslie) [inline en_us/release_notes/source/2015/studio/studio_0804_2015.rst]: Is there a JIRA for this? I hadn't heard of this one, and this seems like the wrong fix if the button doesn't actually work.

Comment (srpearce) [inline en_us/release_notes/source/2015/studio/studio_0804_2015.rst]: Yep - TNL-645

Comment (srpearce) [inline en_us/release_notes/source/2015/studio/studio_0804_2015.rst]: Changed this to be less incriminating:

For drag and drop and custom JavaScript problems, edX has changed the default value of the **Show Answer** setting to **Never**. (TNL-645)

Comment (explorerleslie) [inline en_us/release_notes/source/2015/studio/studio_0804_2015.rst]: @srpearce yes, much better. I read the ticket and it made sense so I like your rewording.

**Comment by mhoeber** [comment]
:+1:

**Pull request merged**